README.md

```diff
@@ -1,8 +1,18 @@
-# PTG
+# PTG ANGEL System
 
-Inital repo for PTG project. Contains the Unity project, the HoloLens 2 research mode visual studio project, and various python scripts to communicate with the HoloLens.
+Inital repo for PTG project.
+Contains the Unity project, the HoloLens 2 research mode visual studio project,
+and various python scripts to communicate with the HoloLens.
 
-# Tool versions used
+This repo contains:
+* Experimental Unity app for transmitting appropriate sensor data off of the
+  HoloLens2 platform
+* ROS 2 system for receiving sensor data, processing analytics, and 
```

Comment:
Super minor, but this seems to be missing the rest of the sentence.

Reply:
Ha I do that sometimes...